endpoints/citrus-http/src/test/java/org/citrusframework/http/message/CookieConverterTest.java

```diff
@@ -1,11 +1,11 @@
 /*
- *    Copyright 2018 the original author or authors
+ *    Copyright 2018-2024 the original author or authors
  *
  *    Licensed under the Apache License, Version 2.0 (the "License");
  *    you may not use this file except in compliance with the License.
  *    You may obtain a copy of the License at
  *
- *        http://www.apache.org/licenses/LICENSE-2.0
+ *        http:// www.apache.org/licenses/LICENSE-2.0
```

Comment:
should do a global pattern match for these:

```suggestion
 *        http://www.apache.org/licenses/LICENSE-2.0
```